bin/netgrasp

```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env python2
 
-from netgrasp.utils import cli
-from netgrasp import netgrasp
-
 import argparse
 
+from netgrasp import netgrasp
+from netgrasp.utils import cli
+
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(
-            prog="netgrasp",
-            description="A passive network observation tool")
+    parser = argparse.ArgumentParser(prog="netgrasp",
+                                     description="A passive network observation tool")
```

Comment:
I'd like to review just the functional changes -- please don't include these sorts of spacing changes/cleanup with functional changes, as it can hide unexpected consequences and makes a thorough review more difficult.